lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -164,3 +164,323 @@ testRule(
 		],
 	}),
 );
+
+testRule({
+	ruleName,
+	config: ['lower'],
+	syntax: 'css-in-js',
+	fix: true,
+	accept: [
+		{
+			code: `
+import styled from 'styled-components';
```

Comment:
I believe can remove `import styled from 'styled-components';` from all the tests as we're explicitly specifying the syntax.